db/schema.rb

```diff
@@ -26,6 +26,12 @@
   add_index "activity_types", ["abbreviation"], name: "index_activity_types_on_abbreviation", unique: true, using: :btree
   add_index "activity_types", ["name"], name: "index_activity_types_on_name", unique: true, using: :btree
 
+  create_table "ar_internal_metadata", primary_key: "key", force: :cascade do |t|
```

Comment:
I think this is left over from the rails upgrade...

Do you really need to add this table to the database for this test change to work?

Reply:
I suggest you revert the changes to this file.

In the future take care with what you are adding to each commit.

Reply:
These have not been addressed.